packages/core/src/components/dropdown/index.js

```diff
@@ -315,15 +317,9 @@ class Dropdown {
         ...options
       };
     }
-    markupTemplates.forEach(template => {
```

Comment:
🤔 this was removed, is it no longer needed or should you be calling `this._renderMarkup`?

Reply:
nope. i re-tested this and figured out that if we wan't to apply new options we do not need to re-render anything. Just set the value and (renderSelected) will be applied and update the option list and (renderOption) also will be invoked